modules/creating-infra-machines-from-compute.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="creating-infra-machines-from-compute_{context}"]
+= Creating infrastructure machines
+
+You can create infrastructure machines by applying a custom label to an existing compute machine.
+
+.Prerequisites
+
+* You have created a compute machine set that has provisioned compute machines.
```

Comment:
Do we need to add; https://github.com/openshift/openshift-docs/pull/30683/files#diff-2901799cc6024aefc0d385a7a42a24f967dc1d1e2b9a4a349b3529f39541e966 as a pre-requisite (should a user have this first)?

Reply:
The module for creating an infra MCP had a verification step at the end:
```
5. Apply the machine config, then verify that the infra-labeled nodes are updated with the configuration file
```

I assumed from that text that it's OK to have the infra nodes first, directly followed by creating the infra MCP.

Reply:
Order technically doesn't matter.  You can "make" the infra nodes first or the MCP first.  If you make the infra nodes and remove the worker label but fail to make the MCP you will find yourself in trouble (MCO won't reconcile the nodes).  For that reason, it would be best practice to make the MCP first, then label your infra nodes then remove the worker label. 

Also, keep in mind, you don't _need_ any extra MachineConfigs for this to work.   So that step 5 verification is a poor method of validation.  I would prefer you do `oc get mcp` to confirm the result set shows your new pool with members in that pool.

Reply:
I'll rearrange these a bit to follow that ordering. That would be a logical sequence of steps :+1:

Reply:
>I would prefer you do `oc get mcp` to confirm the result set shows your new pool with members in that pool.

@briantward if we were creating the infra MCP first, the new pool wouldn't necessarily have any members in the pool to verify, right? Or am I misunderstanding?

Reply:
@yuqi-zhang is that correct (what @briantward says)? I am worried that we will orphan nodes doing this (that's not something we want).

Reply:
I believe Brian's comment is correct, but so long as you have the machineconfigpool created at some point, the node will become un-orphaned. I will double check this.

If that's the only case the node becomes fully orphaned is if you 1. do not create the infra pool and 2. remove the worker label. We should then always document in the order of infra pool first, such that the unmanaged case shouldn't happen

Reply:
I have reordered the procedures to have MCP creation come first.

Reply:
I've verified that creating MCP first or after doesn't matter. It will retroactively "transfer" the corresponding node over to its pool after creation.